apps/web/playwright/bookings-list.e2e.ts

```diff
@@ -95,6 +95,23 @@ test.describe("Bookings", () => {
       await firstGuest.click();
       await expect(titleAndAttendees.locator('[data-testid="unmark-no-show"]')).toBeVisible();
       await expect(titleAndAttendees.locator('[data-testid="mark-no-show"]')).toBeHidden();
+      await webhookReceiver.waitForRequestCount(1);
+      const [request] = webhookReceiver.requestList;
+      const body = request.body;
+      // remove dynamic properties that differs depending on where you run the tests
+      const dynamic = "[redacted/dynamic]";
+      // @ts-expect-error we are modifying the object
+      body.createdAt = dynamic;
+      expect(body).toMatchObject({
+        triggerEvent: "BOOKING_NO_SHOW_UPDATED",
+        createdAt: "[redacted/dynamic]",
+        payload: {
+          message: "x_marked_as_no_show",
+          attendees: [{ email: "first@cal.com", noShow: true, utcOffset: null }],
+          bookingUid: bookingWhereFirstUserIsOrganizer?.uid,
+        },
+      });
+      webhookReceiver.close();
```

Comment:
Decided to add the test in here instead of webhooks to avoid repeating the whole logic.